pubsub/gcp/pubsub/metadata.yaml

```
@@ -0,0 +1,64 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: pubsub.gcp.pubsub
+version: v1
+status: stable
+title: "GCP Pub/Sub"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-gcp-pubsub/
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "gcp"
+metadata:
+  - name: orderingKey
+    description: |
+      The key provided in the request. It’s used when enableMessageOrdering 
+      is set to true to order messages based on such key.   
+    type: string
+    example: “my-orderingkey”
```

Note the quotes used here were not standard ASCII quotes.

```suggestion
    example: '"my-orderingkey"'
```